es5-shim.js

```diff
@@ -329,6 +351,94 @@ var strSplit = call.bind(StringPrototype.split);
 var strIndexOf = call.bind(StringPrototype.indexOf);
 var push = call.bind(array_push);
 
+// ES-5 15.3.4.4
+// http://es5.github.io/#x15.3.4.4
+var hasToStringTagBasicBug = toStr() !== '[object Undefined]' || toStr(null) !== '[object Null]';
+var hasToStringTagLegacyArguments = toStr(function () {
+  return arguments;
+}()) !== '[object Arguments]';
+var hasToStringTagArrayBug = toStr([]) !== '[object Array]';
+var hasToStringTagRegExpBug = toStr(/test/) !== '[object RegExp]';
```

Comment:
just making sure we don't forget to delete this line, unless we can find (and document in a comment) a specific browser that would give `true` here.